pkgs/os-specific/linux/lkvm/default.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenv, fetchgit, libbfd, zlib, libaio, glibc }:
+
+stdenv.mkDerivation {
+  name = "lkvm";
+  version = "2019-10-2525";
```

Comment:
2525 seems wrong. I assume upstream doesn't do releases?

Reply:
Indeed, does not do releases. Updated with date of latest commit. Hope it is fine. Solved all the other requests.